app/controllers/draws_controller.rb

```diff
@@ -183,15 +184,17 @@ def calculate_group_metrics
     @groups_by_size =
       empty_groups_hash.merge(@groups.sort_by { |g| Group.statuses[g.status] }
                                      .group_by(&:size))
+    @groups_by_size.default = 0
+    @group_sizes = @groups_by_size.keys
   end
 
   def calculate_oversub_metrics # rubocop:disable AbcSize
     return unless policy(@draw).oversub_report?
-    zeroed_group_hash = @suite_sizes.map { |s| [s, 0] }.to_h
+    zeroed_group_hash = @group_sizes.map { |s| [s, 0] }.to_h
```

Comment:
I feel like we want to use `@group_sizes + @suite_sizes` here, otherwise we won't show suite sizes with no groups. I think that was originally my point in #278, sorry if it wasn't clear.